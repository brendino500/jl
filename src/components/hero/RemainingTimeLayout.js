import { makeStyles } from '@material-ui/core/styles'
import { Typography } from '@material-ui/core'

const useStyles = makeStyles((theme) => ({
  timeTextLayout: {
    display: 'flex',
    flexDirection: 'column',
    alignItems: 'center',
    justifyContent: 'center',
    margin: '1em',
  },
}))

export default function RemainingTimeLayout({ time, type }) {
  const classes = useStyles()
  return (
    <div className={classes.timeTextLayout}>
      <Typography>{time || '00'}</Typography>
      <Typography>{type}</Typography>
    </div>
  )
}
