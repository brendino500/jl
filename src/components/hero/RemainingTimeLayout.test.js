import renderer from 'react-test-renderer'
import RemainingTimeLayout from './RemainingTimeLayout'

it('RemainingTimeLayout matches snapshot', () => {
  const tree = renderer.create(<RemainingTimeLayout />).toJSON()
  expect(tree).toMatchSnapshot()
})
