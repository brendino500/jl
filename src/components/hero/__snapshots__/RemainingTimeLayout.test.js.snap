// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RemainingTimeLayout matches snapshot 1`] = `
<div
  className="makeStyles-timeTextLayout-1"
>
  <p
    className="MuiTypography-root makeStyles-timeNumbers-2 MuiTypography-body1"
  >
    00
  </p>
  <p
    className="MuiTypography-root makeStyles-timeText-3 MuiTypography-body1"
  />
</div>
`;
